backend/app/data_processing/gesture.py

```diff
@@ -99,3 +103,43 @@ def get_instrument_sliders(gestures, gesture_param):
             coordinates_sum += coordinate['x'] + coordinate['y']
         result.append(coordinates_sum)
     return result
+
+
+def generate_samples_from_gesture(gesture, canvas):
```

Comment:
This works great. The result is a very clear mapping between the gesture input and the output audio.